src/main/java/seedu/address/logic/commands/leaderboardcommand/LeaderboardCommand.java

```diff
@@ -12,11 +16,28 @@
 public abstract class LeaderboardCommand extends Command {
 
     public static final String COMMAND_WORD = "leaderboard";
+    public static final String MESSAGE_NO_TEAM = "Leaderboard cannot be generated: no Teams present in Alfred. "
+            + "\nAdd more Teams to view Leaderboard.";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": displays the leaderboard as it currently stands. \n"
```

Comment:
This can be more specific, like saying "ranks team in descending order according to score" etc.